src/components/RadialSlider/hooks/useRadialSlider.ts

```diff
@@ -22,11 +22,31 @@ const useRadialSlider = (props: RadialSliderHookProps) => {
 
   const centerValue = Math.round((max - min) / 2) as number;
 
+  //For default variant in radial slider
+  const isRadialSliderVariant = variant === Constants.radialSlider;
+
+  //For radial-circle-slider variant
   const isRadialCircleVariant = variant === Constants.radialCircleSlider;
 
+  //For speedometer-marker variant
+  const isMarkerVariant = variant === Constants.speedoMeterMarker;
+
+  //For speedometer variant
+  const isSpeedoMeterVariant = variant === Constants.speedometer;
+
   const radianValue = isRadialCircleVariant ? 0.057 : openingRadian;
 
-  const isMarkerVariant = variant === Constants.speedoMeterMarker;
+  useEffect(() => {
+    if (isMarkerVariant)
+      if (min < 0) {
+        throw 'Negative number is not allowed';
+      } else if (max < 0) {
+        throw 'Negative number is not allowed';
+      }
+    if (max < min) {
+      throw 'max value should be more than min';
```

Comment:
@parth-simform Update wording as follows and also formate code
`max value should be greater than min`

Reply:
Resolved